common/static/common/templates/discussion/new-post-menu-category.underscore

```diff
@@ -1,4 +1,3 @@
-<li role="menuitem" class="topic-menu-item">
-    <span class="topic-title"><%- text %></span>
-    <ul role="menu" class="topic-submenu"><%= entries %></ul>
-</li>
+<optgroup label="<%- text %>">
+    <%= entries %>
```

Comment:
For safe templates, I think you should use:

    <%= HtmlUtils.ensureHtml(entries) %>